pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -446,6 +446,378 @@
         "alpine.3.4.3-x64": {
             "#import": [ "alpine.3.4.3", "alpine.3-x64" ]
         },
+
```

Comment:
We should think about generating this file during our build rather than all of this duplication.  This seems way too prone to type-o's/inconsistencies.